samples/peripheral/radio_test/README.rst

```diff
@@ -0,0 +1,149 @@
+At any time during the tests, you can set the radio parameters, such as output power, bit rate, and channel.
+You can also set the time on each channel in the sweep mode, in steps of 1 millisecond to 99 milliseconds, every 1 millisecond.
+The sample also allows you to send a data pattern to another board.
+
+The sample starts with enabling the high frequency crystal oscillator and configuring CLI.
+Running a console command causes the appropriate action.
+
+.. note::
+   For the IEEE 802.15.4 mode, the start channel and the end channel must be within the channel range of 11 to 26.
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF5340DK|
+
+    * Network Core
+
+  * |nRF52840DK|
+  * |nRF52DK|
+
+* One of the following testing devices:
+
+  * Spectrum analyzer
+  * Another board with the same sample
+  * Another board running RSSI Viewer available in the `nRF Connect for Desktop`_ application
+
+
+.. _radio_test_ui:
+
+User interface
+**************
+.. list-table:: Main CLI commands (in alphabetical order)
+   :header-rows: 1
+
+   * - Command
+     - Argument
+     - Description
+   * - cancel
+     -
+     - Cancel the sweep or the carrier.
+   * - data_rate
+     - <sub_cmd>
+     - Set the data rate.
+   * - end_channel
+     - <channel>
+     - End the channel for the sweep.
+   * - output_power
+     - <sub_cmd>
+     - Output power set.
+   * - parameters_print
+     -
+     - Print current delay, channel, and other parameters.
+   * - print_rx
+     -
+     - Print the received RX payload.
+   * - start_channel
+     - <channel>
+     - Start the channel for the sweep or the channel for the constant carrier.
+   * - start_duty_cycle_modulated_tx
+     - <duty_cycle>
+     - Duty cycle in percent (two decimal digits, between 01 and 99).
+   * - start_rx
+     -
+     - Start RX.
+   * - start_rx_sweep
+     -
+     - Start the RX sweep.
+   * - start_tx_carrier
+     -
+     - Start the TX carrier.
+   * - start_tx_modulated_carrier
+     - <packet_num>
+     - Start the modulated TX carrier (continuous TX mode is used if no argument is provided).
+   * - start_tx_sweep
+     -
+     - Start the TX sweep.
+   * - time_on_channel
+     - <time>
+     - Time on each channel (between 1 ms and 99 ms).
+   * - toggle_dcdc_state
+     - <state>
+     - Toggle DC/DC converter state.
+   * - transmit_pattern
+     - <sub_cmd>
+     - Set transmission pattern.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/peripheral/radio_test`
+
+.. include:: /includes/build_and_run.txt
+
+.. _radio_test_testing:
+
+Testing
+=======
+
+After programming the sample to your board, test it by performing the following steps:
+
+1. Connect the board to the computer using a USB cable.
+   The board is assigned a COM port (Windows) or ttyACM device (Linux), which is visible in the Device Manager.
+#. |connect_terminal|
+#. Type any command described in the :ref:`radio_test_ui`.
```

Comment:
This is not helpful for new users.
We should give a set of clear instructions, and then we can end with something like "you can also enter any command described in ...".
But first we should give the users instructions for 3-5 useful commands:
"Type XXX to configure something and then startXXX to start doing something. Observe that ...
Type YYY ..."

If this gets difficult because we have different methods of checking the result (spectrum analyzer/RSSI Viewer or a second board), we should either have two sets of testing instructions or give a general explanation of the different ways of testing first.